README.md

```diff
@@ -83,4 +83,152 @@ To make sure Docker automatically starts on boot, run
+
+> JWT generates, access and refresh tokens for Users wanting to access information from the API
+
+>The command `permission_classes = (permissions.IsAuthenticated,)` located in `views.py` ensure that in order to access information of the API you must have logged in with your access token.
+
+•	In order to get a JWT Token, you have to register a new user. 
+
+•	When a new user is created, email verification is sent to their email. You can try this by creating a user for yourself
+
+•	After the User has been registered and has verified their email, they are able to login to the API (View Procedure to Login to the API Below)
+
+•	Upon Logging In, when correct credentials are entered, you receive a 201 response from the API with the entered username, email and two tokens generated.
+
+
+After you receive the tokens by logging in. You will notice that there is one that states, "Refresh" and another one that states "Access". 
+
+•	The Refresh token has a life span of 30 days (which can be changed in `settings.py` ) and is used to generate a new token when required.
+
+•	The Acces token enables the users to have direct access to the API, it has a life span of 20 minutes (which can be changed in `settings.py` )
+
+•	When you have these tokens you can then login into the API interface and access all features.
+
+## How To Login A User
+
+> To be able to use the API and access all the information, a user has to login with generated access token
+
+To use the login feature and fully have access to the API:
+	
+    • Run these command in terminal
+    • export DB_PASSWORD=arbitrarypassword
+    • docker-compose run web python3 manage.py migrate
+    • docker-compose up
+    • On your web browser access the API using http://localhost:8000
+
+
+• When the API has loaded up, you would see an interphace with various function as shown in the image below
+ 
+![Sample Image 1](https://github.com/dsc-umass/lmvp/blob/f8f0b74ace33f7e4d4409bb473d3fa11d4b59b34/docs/images/SW-1.png)
+
+• Click the button that states `(Authorize)`
+
+• Upon clicking Authorize you will see a pop-up requesting for API Key
+
+![Sample Image 2](https://github.com/dsc-umass/lmvp/blob/f8f0b74ace33f7e4d4409bb473d3fa11d4b59b34/docs/images/SW-2.png)
+
+• At this point enter Bearer followed by the ACCESS token and click Authorize `Example:(Bearer **************)`. This will give you full access to the API and its functions
+
+## How To Use Login Feature For User And Get Tokens
+
+>To be able to use the API and access all the information, a user has to login to access the login tokens
+
+To use the login feature for a user follow the following steps (Beware test Users have already been created, details below):
+
+    • Run these command in terminal
+    • export DB_PASSWORD=arbitrarypassword
+    • docker-compose run web python3 manage.py migrate
+    • docker-compose up
+    • On your web browser access the API using http://localhost:8000

```

Comment:
Only code or terminal commands should be in code blocks, and bullet points should not be used in code blocks. I also suggest this change for other code blocks in the README.

```suggestion
    $ export DB_PASSWORD=arbitrarypassword
    $ docker-compose run web python3 manage.py migrate
    $ docker-compose up
In your web browser, access the API using http://localhost:8000

```